src/Task/SentryTestConnectionTask.php

```diff
@@ -49,7 +50,7 @@ public function run($request = null): void
         $logger = Injector::inst()->createWithArgs(Logger::class, ['error-log'])
             ->pushHandler(SentryHandler::create());
 
-        foreach (array_keys(Logger::getLevels()) as $name) {
+        foreach (Level::NAMES as $name) {
```

Comment:
Are you sure this works? Shouldn't this be `Level::VALUES` or `Level::cases()`?

Reply:
Yup I tested it out and made sure it was working. The previous iteration was specifically grabbing the array_keys of getLevels to get the labels, using `Level::VALUES` attempts to call `$logger->100()` instead of `$logger->debug()` which is why I've used `Level::NAMES`